server/src/main/java/org/apache/druid/server/http/TaskLockResponse.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server.http;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.druid.metadata.TaskLockInfo;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class TaskLockResponse
+{
+  private final Map<String, List<TaskLockInfo>> taskLocks;
+
+  @JsonCreator
+  public TaskLockResponse(
+      @JsonProperty("taskLocks") final Map<String, List<TaskLockInfo>> taskLocks
+  )
+  {
+    this.taskLocks = taskLocks;
+  }
+
+  @JsonProperty
+  public Map<String, List<TaskLockInfo>> getTaskLocks()
```

Comment:
Nit: either call it `datasourceToTaskLocks` (or similar) or add a javadoc.

Reply:
Done